src/SFML/Window/Win32/WindowImplWin32.cpp

```diff
@@ -1174,6 +1353,48 @@ LRESULT CALLBACK WindowImplWin32::globalOnEvent(HWND handle, UINT message, WPARA
     return DefWindowProcW(handle, message, wParam, lParam);
 }
 
+////////////////////////////////////////////////////////////
+void WindowImplWin32::prepareTouch()
+{
+    static bool prepared = false;
+    if (!prepared)
+    {
+        prepared = true;
+
+#if WINVER < 0x0601
+        RegisterTouchWindow = reinterpret_cast<RegisterTouchWindowFuncType>(GetProcAddress(user32Dll, "RegisterTouchWindow"));
+
+        touchEnabled = RegisterTouchWindow != nullptr;
+
+        // If we've got touch support, load the other procs
+        if (touchEnabled)
+        {
+            CloseTouchInputHandle = reinterpret_cast<CloseTouchInputHandleFuncType>(GetProcAddress(user32Dll, "CloseTouchInputHandle"));
+            GetTouchInputInfo = reinterpret_cast<GetTouchInputInfoFuncType>(GetProcAddress(user32Dll, "GetTouchInputInfo"));
+
+            // Reset touch IDs
+            for (int i = 0; i < 10; ++i)
+                touchIDs[i] = static_cast<DWORD>(-1);
+        }
```

Comment:
This code handles OSes older than Windows 7 which I think we're save to ignore at this point.

Reply:
Have to admit I'm a bit out of the loop here, but as far as I remember, without the legacy concern, large parts of these changes should be able to be simplified now (read: just use regular headers and let the linker pull the dependencies in).

The section you highlighted was for Windows XP and Vista and I don't think I was even able to properly test this apart from a VM with faked touch inputs. All the dependencies should be there when compiling for Windows 7 and later, so it should be fine to drop all the manually inserted header sections, too.